### ISSUE TITLE
Appearance: system theme doesn't work on Ubuntu

### ISSUE THREAD

**Issue by osmaczko** [status-im/status-desktop#5820]
https://user-images.githubusercontent.com/33099791/169365641-508e333b-97b8-4bc3-8c44-bc4da7829ff3.mp4

**Comment by osmaczko** [comment]
I've investigated the issue a little bit deeper and haven't found any solution which does not require calls to OS native methods.

The current solution, based on the [window color heuristic algorithm](https://github.com/status-im/status-desktop/blob/8685ad92c9ac9377ca75480c12d6c3a238148d38/ui/main.qml#L199-L207) works fine for macOS (and Windows?) but not for Linux (GNOME) - `window.hslLightness` doesn't change together with system theme. I've also noticed that `changeThemeFromOutside` is not triggered because neither of the observed events from `OSThemeEvent::eventFilter` is captured.

Qt is currently working on a cross-platform deterministic solution: https://bugreports.qt.io/browse/QTBUG-97910 which will be probably delivered in some upcoming Qt6 releases.

---
@saledjenic as you were the one working on the current solution, maybe you have some other ideas on how to work around the problem until Qt provides the solution?
